<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExerciseDialog</class>
 <widget class="QDialog" name="ExerciseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditDescription"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Target Muscle:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboMuscles">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Chest </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Back</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Biceps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Triceps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shoulders</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Forearms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Abs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Quads</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hamstrings</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Glutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Calves</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Full Body</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioBodyweight">
     <property name="text">
      <string>Bodyweight</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioWeighted">
     <property name="text">
      <string>Weighted</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExerciseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExerciseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
